### PR TITLE
[Do Not Merge]RIA-2775: Force case progression to case under reveiw.

### PR DESCRIPTION
**Pull request by martin-bucinskas**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-2775

### Change description ###
 - A new case note is added to the appeal on the force case progression to case under review event.
 - A confirmation has been created for the event.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by iguannago** [comment]
so when we force the case from caseBuilding to case under review the only things that happens are: 
1. case changes to the case under review state
2. a caseNote is added
3. a notification is sent
But wasn't there any other logic to apply? I mean, does not any other field in the case have to change? or anything like that?

**Comment by iguannago** [comment]
revert this commit 626d2c7e85ffe5d4f11d644e4ac4042bf8fa6e42 before merging it.

File: charts/ia-case-api/values.preview.template.yaml [hunk 1]
```diff
@@ -9,9 +9,9 @@ java:
     POSTGRES_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
     POSTGRES_CONNECTION_OPTIONS: ""
     IA_CASE_DOCUMENTS_API_URL: "http://ia-case-documents-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/"
-    IA_CASE_NOTIFICATIONS_API_URL: "http://ia-case-notifications-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/"
     IA_TIMED_EVENT_SERVICE_URL: "http://ia-timed-event-service-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/"
     IA_TIMED_EVENT_SERVICE_ENABLED: "false"
+    IA_CASE_NOTIFICATIONS_API_URL: "http://ia-case-notifications-api-pr-238.service.core-compute-preview.internal/"
```
Comment (makowalski) [inline charts/ia-case-api/values.preview.template.yaml:14]: Could you revert that change? I am going to merge this PR soon

**Comment by makowalski** [comment]
closing this PR to start new pod within new PR

**Pull request closed without merging**